sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.institutions.controller.getCompletedApplications.e2e-spec.ts

```diff
@@ -0,0 +1,260 @@
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+      collegeFLocation,
+    );
+  });
+
+  it(`Should return all completed application offering change requests for the location in ${FieldSortOrder.ASC} order of name when requested.`, async () => {
+    // Arrange
+
+    // Student 1 has a completed application to the institution.
+    await saveFakeApplicationDisbursements(
+      db.dataSource,
+      { institutionLocation: collegeFLocation },
+      {
+        applicationStatus: ApplicationStatus.Completed,
+      },
+    );
+    // Student 2 has an in-progress with student application offering a change request for the institution.
+    await saveFakeApplicationOfferingRequestChange(db.dataSource, {
+      institutionLocation: collegeFLocation,
+    });
+    // Student 3 has an approved application offering change request for the institution.
+    const applicationOfferingChange3 =
+      await saveFakeApplicationOfferingRequestChange(
+        db.dataSource,
+        {
+          institutionLocation: collegeFLocation,
+        },
+        {
+          applicationOfferingChangeRequestStatus:
+            ApplicationOfferingChangeRequestStatus.Approved,
+        },
+      );
+    // Student 4 has an in progress with student application offering change request with a different institution.
+    await saveFakeApplicationOfferingRequestChange(db.dataSource);
+    // Student 5 has a declined by student application for the institution, that have an approved application offering change request.
+    const applicationOfferingChange5 =
+      await saveFakeApplicationOfferingRequestChange(
+        db.dataSource,
+        {
+          institutionLocation: collegeFLocation,
+        },
+        {
+          applicationOfferingChangeRequestStatus:
+            ApplicationOfferingChangeRequestStatus.DeclinedByStudent,
+        },
+      );
+    // Student 6 has a declined by student application for the institution, that have an approved application offering change request.
+    const applicationOfferingChange6 =
+      await saveFakeApplicationOfferingRequestChange(
+        db.dataSource,
+        {
+          institutionLocation: collegeFLocation,
+        },
+        {
+          applicationOfferingChangeRequestStatus:
+            ApplicationOfferingChangeRequestStatus.DeclinedBySABC,
+        },
+      );
+
+    applicationOfferingChange3.assessedDate = new Date();
+    applicationOfferingChange5.studentActionDate = new Date();
+    applicationOfferingChange6.assessedDate = new Date();
+
+    await db.applicationOfferingChangeRequest.save([
+      applicationOfferingChange3,
+      applicationOfferingChange5,
+      applicationOfferingChange6,
+    ]);
+
+    const application3 = applicationOfferingChange3.application;
+    const application5 = applicationOfferingChange5.application;
+    const application6 = applicationOfferingChange6.application;
+
+    application3.applicationNumber = "1000000000";
```

Comment:
Maybe the application numbers can be assigned in the opposite order the records were created to ensure that the order was enforced. Just a suggestion.